.github/workflows/ci.yml

```diff
@@ -450,10 +450,19 @@ jobs:
           - config: apache-hive3
             suite: suite-hms-only
             jdk: 11
-          # suite-1 does not contain Kerberos/encrypted Hive environments
+          # run all HDP3 suites on JDK 17
           - config: hdp3
             suite: suite-1
             jdk: 17
+          - config: hdp3
+            suite: suite-2
+            jdk: 17
+          - config: hdp3
+            suite: suite-3
```

Comment:
All of them? Why?

Reply:
Just a test. Now when Kerberos is fixed for Hive/Hadoop I can see whether something else is broken